tests/by-util/test_ls.rs

```diff
@@ -3522,3 +3522,126 @@ fn test_ls_perm_io_errors() {
         .code_is(1)
         .stderr_contains("Permission denied");
 }
+
+#[test]
+fn test_ls_dired_incompatible() {
+    let scene = TestScenario::new(util_name!());
+
+    scene
+        .ucmd()
+        .arg("--dired")
+        .fails()
+        .code_is(1)
+        .stderr_contains("--dired requires --format=long");
+}
```

Comment:
This test seems to be incorrect. At least with GNU ls 9.3, running `ls --dired` doesn't fail. It simply ignores the `--dired` option.

Reply:
Yeah, but it is a GNU issue. I will fix it there too.

Reply:
here it is:
https://lists.gnu.org/archive/html/coreutils/2023-09/msg00050.html